uxarray/grid/grid.py

```diff
@@ -1749,6 +1751,29 @@ def get_kd_tree(
 
         return self._kd_tree
 
+    def get_spatialhash(
```

Comment:
```suggestion
    def get_spatial_hash(
```

I think this name is a better fit. It should fix the notebook too, since it's using this syntax.

![image](https://github.com/user-attachments/assets/4a459a24-257e-4d1b-a522-37d0e4162673)